src/camvidlog2/yoloe/ai.py

```
@@ -0,0 +1,214 @@
+    Returns:
+        np.ndarray: Preprocessed image of shape (1, 3, 640, 640), dtype float32, values in [0, 1].
+            - Channel order: [R, G, B]
+            - Pixel values normalized to [0, 1]
+            - Suitable for ONNX model input
+    """
+    frm = cv2.resize(frame, (640, 640), interpolation=cv2.INTER_LINEAR)
+    frm = cv2.cvtColor(frm, cv2.COLOR_BGR2RGB)
+    frm = frm.transpose(2, 0, 1)  # (3, 640, 640)
+    frm = frm.reshape(1, 3, 640, 640)
+    frm = (frm / 255.0).astype(np.float32)
+    return frm
```

Ruff (0.11.9)

Unnecessary assignment to `frm` before `return` statement

Remove unnecessary assignment

(RET504)